flint.ui/src/components/Button/Button.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <button type="button" @click="onClick" :style="btnStyles">
```

Comment:
Replace this with
```
 <button type="button" :style="btnStyles" @click="onClick">
    <slot></slot>
  </button>
  ```
  
  Basically, we want to use Slot in order to have `<Button> Click me </Button>` usage. You can read more about it though.

Reply:
fixed